ts/main.rs

```diff
@@ -0,0 +1,20 @@
+pub mod tests;
+use tests::TestsSuite;
+
+fn setup() {
+   println!("----------------- Setup -----------------")
```

Comment:
In this case it ok to use `println` and not the logger? Not pointing out, just asking to learn how this is done in rust.

Reply:
![image](https://user-images.githubusercontent.com/5485589/140290526-d27af073-3da7-4af8-b797-f32db7e4a5b6.png)
No, in the setup & teardown functions, we can put sth like setup logger served for testing. Loading test data at `setup()` step, and do clean up on `teardown()`. Its more like my demo about tests suite.